friktion/friktion/bid_details.py

```diff
@@ -10,8 +10,8 @@
 class BidDetails:
     signer_wallet: PublicKey
     order_id: int
-    bid_size: int  # 1e9
-    bid_price: int  # 1e9
+    bid_size: int  # 1, the smart contract always initializes option token with 0 decimals
+    bid_price: int  # number of bid tokens per offer token. normalization factor of 1.
```

Comment:
not sure if I understood the comments. In the slack thread you said it should be 1e6

Reply:
answered on slack